### PR TITLE
Fix bugs in the Console app

### PR DESCRIPTION
**Pull request by thivi**
Resolves https://github.com/wso2/product-is/issues/9719
Resolves https://github.com/wso2/product-is/issues/9729
Resolves https://github.com/wso2/product-is/issues/9722
Resolves https://github.com/wso2/product-is/issues/8723
Resolves https://github.com/wso2/product-is/issues/8724
Resolves https://github.com/wso2/product-is/issues/9728
Resolves https://github.com/wso2/product-is/issues/8726
Resolves https://github.com/wso2/product-is/issues/9678

### Password strength meter
image

### Email template type search feature
image

### Name of the resource being created in the header of the wizard
image

### CODE REVIEW

File: apps/console/src/features/email-templates/pages/email-template-types-page.tsx [hunk 1]
```diff
@@ -146,8 +166,52 @@ const EmailTemplateTypesPage: FunctionComponent<EmailTemplateTypesPagePropsInter
...
+        setFilteredEmailTemplateTypes(emailTemplateTypes);
+    };
+
+    /**
+     * Handle sort strategy change.
+     *
+     * @param {React.SyntheticEvent<HTMLElement>} event.
+     * @param {DropdownProps} data.
+     */
+    const handleSortStrategyChange = (event: React.SyntheticEvent<HTMLElement>, data: DropdownProps) => {
```
Comment (brionmario) [inline apps/console/src/features/email-templates/pages/email-template-types-page.tsx:204]: Missing return type. Please check other places as well.

```suggestion
    const handleSortStrategyChange = (event: React.SyntheticEvent<HTMLElement>, data: DropdownProps): void => {
```

File: apps/console/src/features/applications/components/application-list.tsx [hunk 2]
```diff
@@ -352,7 +352,8 @@ export const ApplicationList: FunctionComponent<ApplicationListPropsInterface> =
             );
         }
 
-        if (list?.totalResults === 0) {
+        if (list?.totalResults === 0
+            || (list?.applications?.find(app => app.name === "wso2carbon-local-sp") && list?.totalResults === 1)) {
```
Comment (brionmario) [inline apps/console/src/features/applications/components/application-list.tsx:356]: Shall we get `wso2carbon-local-sp` to a constant. There is one more usage. Would be better to refactor that as well.

**Review by brionmario — APPROVED** [review]

File: apps/console/src/features/email-templates/pages/email-template-types-page.tsx (hunk 1 above)
Comment (thivi) [reply apps/console/src/features/email-templates/pages/email-template-types-page.tsx]: Resolved by 7990967

File: apps/console/src/features/applications/components/application-list.tsx (hunk 2 above)
Comment (thivi) [reply apps/console/src/features/applications/components/application-list.tsx]: Resolved by d6bf687

**Pull request merged**